addons/mail/static/tests/helpers/mock_server/models/mail_tracking_value.js

```diff
@@ -71,10 +72,17 @@ patch(MockServer.prototype, {
                 values["old_value_integer"] = initialValue ? 1 : 0;
                 values["new_value_integer"] = newValue ? 1 : 0;
                 break;
-            case "monetary":
+            case "monetary": {
                 values[`old_value_float`] = initialValue;
                 values[`new_value_float`] = newValue;
+                let currencyField = field.currency_field;
+                // see get_currency_field in python fields
+                if (!currencyField && "currency_id" in record) {
+                    currencyField = "currency_id";
+                }
+                values[`currency_id`] = record[currencyField];
```

Comment:
This notation is also okay to do.
```suggestion
                values.currency_id = record[currencyField];
```